docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -157,131 +349,211 @@ Triggers automatically have the Storm variable ``$auto`` populated when they run
 
-As such, cron is most appropriate for automating things such as non-urgent tasks, tasks that only need to run intermittently, or resource-intensive tasks that should run during off-hours.
+- Creating a cron job will create an associated ``syn:cron`` runtime node (runt node). While runt nodes
+  (:ref:`gloss-runt-node`) are typically read-only, ``syn:cron`` nodes include ``:name`` and ``:doc``
+  secondary properties that can be set and modified via Storm (or configured via :ref:`gloss-optic`).
+  This allows you to manage cron jobs by giving them meaningful names and descriptions. Changes to these
+  properties will persist even after a Cortex restart.
 
-.. NOTE::
-  When scheduling jobs, cron interprets all times as UTC.
+- ``syn:cron`` nodes can be lifted, filtered, and pivoted across just like other nodes. However, they
+  cannot be created or modified (e.g., using Storm's data modification / edit syntax) except in the
+  limited ways described above.
 
-**Variables:**
+- The creator (owner) of a cron job can be modified (with appropriate permissions) using the Storm
+  :ref:`stormlibs-lib-cron` library and :ref:`stormprims-storm-cronjob-f527` primitive. For example:
+  
+  ::
+  
+    $cron=$lib.cron.get(<cron_iden>) $cron.set(user, <new_user_iden>)
 
-Cron jobs automatically have the Storm variable ``$auto`` populated when they run. The ``$auto`` variable is a dictionary which contains the following keys:
+
+Variables
+~~~~~~~~~
+
+Cron jobs automatically have the Storm variable ``$auto`` populated when they run. The ``$auto`` variable
+is a dictionary which contains the following keys:
 
   ``$auto.iden``
-    The identifier of the Cron job.
+    The identifier of the cron job.
 
   ``$auto.type``
-    The type of automation. For a Cron job this value will be ``cron``.
+    The type of automation. For a cron job this value will be ``cron``.
 
+Syntax
+~~~~~~
 
-**Syntax:**
+Cron jobs are created, modified, viewed, enabled, disabled, and deleted using the Storm ``cron.*``  commands.
+See the :ref:`storm-cron` command in the :ref:`storm-ref-cmd` document for details.
 
-Cron jobs are created, modified, viewed, enabled, disabled, and deleted using the Storm ``cron.*``  commands. See the :ref:`storm-cron` command in the :ref:`storm-ref-cmd` document for details.
+In :ref:`gloss-optic`, cron jobs can also be managed through the :ref:`gloss-admin-tool`.
 
 .. NOTE::
-  Cron jobs can be modified after they are created, but modification is limited to updating the Storm query executed by the job. If other aspects of a job need to be changed (such as its schedule for execution), the job must be deleted and re-created.
+  
+  Once a cron job is created, you can modify many of its properties (such as its name and description, or
+  the Storm associated with the job). However, you cannot modify other aspects of the job, such as its
+  schedule. To change those conditions, you must delete (or disable) and re-create the cron job.
```

Comment:
same comment here as for triggers, disabling and creating a new one doesn't really feel like "changing" to me